product_docs/docs/bdr/4/conflicts.mdx

```diff
@@ -651,191 +642,189 @@ node.
-requires clocks to be in sync to work correctly, or to be within a
+recorded on the host the transaction originates from. This
+requires clocks to be in sync to work correctly or to be within a
 tolerance of the fastest message between two nodes. If this
-is not the case, conflict resolution will tend to favour the node that
-is further ahead. Clock skew between nodes can be managed using the
+isn't the case, conflict resolution tends to favor the node that's
+further ahead. You can manage clock skew between nodes using the
 parameters `bdr.maximum_clock_skew` and `bdr.maximum_clock_skew_action`.
 
-Row origins are only available if track_commit_timestamp = on.
+Row origins are available only if `track_commit_timestamp = on`.
 
-Conflicts are initially detected based upon whether the replication
-origin has changed or not, so conflict triggers will be called in
-situations that may turn out not to be actual conflicts. Hence, this
-mechanism is not precise since it can generate false positive conflicts.
+Conflicts are initially detected based on whether the replication
+origin changed, so conflict triggers are called in
+situations that might turn out not to be conflicts. Hence, this
+mechanism isn't precise, since it can generate false-positive conflicts.
 
 Origin info is available only up to the point where a row is frozen.
-Updates arriving for a row after it has been frozen will not raise
-a conflict, so will be applied in all cases. This is the normal case
-when we add a new node by bdr_init_physical, so raising conflicts
-would cause many false positive cases in that case.
+Updates arriving for a row after it was frozen don't raise
+a conflict so are applied in all cases. This is the normal case
+when adding a new node by `bdr_init_physical`, so raising conflicts
+causes many false-positive results in that case.
 
-When a node that has been offline for some time reconnects and
-begins sending data changes, this could potentially cause divergent
-errors if the newly arrived updates are actually older than the
-frozen rows that they update. Inserts and Deletes are not affected by this situation.
+When a node that was offline reconnects and
+begins sending data changes, this can cause divergent
+errors if the newly arrived updates are older than the
+frozen rows that they update. Inserts and deletes aren't affected by this situation.
 
-Users are advised to not leave down nodes for extended outages,
-as discussed in [Node Restart and Down Node Recovery](nodes).
+We suggest that you don't leave down nodes for extended outages,
+as discussed in [Node restart and down node recovery](nodes).
 
-On EDB Postgres Extended Server and EDB Postgres Advanced Server, BDR will
-automatically hold back the freezing of rows while a node is down to handle
-this situation gracefully without the need for changing parameter settings.
+On EDB Postgres Extended Server and EDB Postgres Advanced Server, BDR
```

Comment:
Obviously, here it would still stay as server as it's the product name, not a node reference.